### ISSUE TITLE
Ipynb report not generated

### ISSUE THREAD

**Issue by abradle** [choderalab/yank#1053]
Having completed a yank run - when I run:

yank report -y free-energy.yaml -o report 

it appears to run - but then report directory is empty:

2018-07-27 09:28:19,042: Setting OpenCL platform to use precision model 'mixed'.
Rendering notebook as a .ipynb file...
Processing notebook now, this may take a while...
2018-07-27 09:28:22,649: Executing notebook with kernel: python3

The full directory with results and input is here:
https://drive.google.com/open?id=15lpbkSbWQw5J7ybIN_eHTlQ0O7xGfj-c

Any help would be gratefuly received.

**Comment by Lnaden** [comment]
@abradle I cannot replicate this behavior locally. If I run the command on the files provide

`yank analyze report -y free-energy.yaml -o report`

I get the correct behavior with YANK 0.23.2. What version of YANK are you using?

**Comment by abradle** [comment]
That's weird - I'm on 0.23.2 too - or at least thats what selftest.  

```
yank selftest
```
```
YANK Selftest
-------------
Yank Version 0.23.2 
```
```
yank analyze report -y free-energy.yaml -o report
```
```
2018-07-27 14:43:35,795: Setting OpenCL platform to use precision model 'mixed'.
Rendering notebook as a .ipynb file...
Processing notebook now, this may take a while...
2018-07-27 14:43:37,376: Executing notebook with kernel: python3
```
and then empty report dir.

No massive issue - particular if you can't reproduce - pretty hard to debug...
I'm running on conda 4.5.8 on RHEL7 or conda 4.5.4 on Ubuntu
with python 3.6.5.

**Comment by Lnaden** [comment]
Can you do a `ls -aR` on the directory you are executing this from after that command runs? There was a bug I fixed where the file name was not being written correctly, but that should be fixed with 23.2

**Comment by abradle** [comment]
```
ls -aR
.:
.  ..
```

**Comment by abradle** [comment]
and from one up

```
ls -aR
.:
.  ..  free-energy.yaml  MURD-x0373_params.mol2  no_buffer_altlocs.pdb	report	sams-twostage-harmonic-dense

./report:
.  ..

./sams-twostage-harmonic-dense:
.  ..  experiments  setup

./sams-twostage-harmonic-dense/experiments:
.  ..  analysis.yaml  complex_checkpoint.nc  complex.nc  experiments.log  experiments.yaml  solvent_checkpoint.nc  solvent.nc

./sams-twostage-harmonic-dense/setup:
.  ..  molecules  setup.log  systems

./sams-twostage-harmonic-dense/setup/molecules:
.  ..  lig

./sams-twostage-harmonic-dense/setup/molecules/lig:
.  ..  lig.frcmod  lig.gaff.mol2

./sams-twostage-harmonic-dense/setup/systems:
.  ..  prot-lig

./sams-twostage-harmonic-dense/setup/systems/prot-lig:
.  ..  complex.inpcrd  complex.leap.in	complex.leap.log  complex.pdb  complex.prmtop  solvent.inpcrd  solvent.leap.in	solvent.leap.log  solvent.pdb  solvent.prmtop
```

**Comment by Lnaden** [comment]
Hmm. I cannot replicate this locally, and it does not look like its writing the file to `.{extension} anywhere. I'll see if I can try on a couple different OS's since I rely on the `os` module to make the paths

**Comment by jchodera** [comment]
@lnaden: Could this be because this is a single experiment (which is placed directly in `{calcname}/experiments/` directly instead of in a named experiment subdirectory) instead of one of multiple experiments?

I'm testing this locally right now as well.

**Comment by Lnaden** [comment]
No, because setting the `-y` flag forces `-o` to target an existing directory. The logic it follows is the same no matter how many things are passed in.
